example/demo_generator/src/admin/server.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2025, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import path from 'path';
+
+import setupServer from 'evolution-backend/lib/apps/participant/index';
```

Comment:
This import should be the one from admin (and no need to put index, as it is implied)